api/renter_test.go

```diff
@@ -1872,31 +1899,19 @@ func TestContractorHostRemoval(t *testing.T) {
 		t.Fatal(err, "::", rf.Files[0].Redundancy)
 	}
 
-	// Check that the amount of data in each contract has remained at the
-	// correct amount - just one sector each.
-	err = st.getAPI("/renter/contracts", &rc)
-	if err != nil {
-		t.Fatal(err)
-	}
-	for _, contract := range rc.Contracts {
-		if contract.Size != modules.SectorSize {
-			t.Error("Each contrat should have 1 sector:", contract.Size, contract.ID)
-		}
-	}
-
 	// Try again to download the file we uploaded. It should still be
 	// retrievable.
 	downloadPath3 := filepath.Join(st.dir, "test-downloaded-verify-3.dat")
 	err = st.stdGetAPI("/renter/download/test?destination=" + downloadPath3)
 	if err != nil {
-		t.Log("FINAL DOWNLOAD HAS FAILED:", err)
+		t.Error("Final download has failed:", err)
 	}
 	downloadBytes3, err := ioutil.ReadFile(downloadPath3)
 	if err != nil {
-		t.Log(err)
+		t.Error(err)
 	}
 	if !bytes.Equal(downloadBytes3, origBytes) {
-		t.Log("downloaded file and uploaded file do not match")
+		t.Error("downloaded file and uploaded file do not match")
```

Comment:
I think this was logged before because it would fail inexplicably on appveyor and we never knew why. I changed it to `t.Error` so that we get a proper failure, and hopefully one of the machines we have ssh access to will eventually fail and we can see the logs.